src/table/use-sticky-scrollbar.ts

```diff
@@ -138,29 +137,13 @@ export function useStickyScrollbar(
           );
         }
       });
+      // Scrollbar width must be in sync with wrapper width.
+      observer.observe(wrapperRef.current);
```

Comment:
This replaces the global resize observer from below and is better as it also detects container width changes not caused by window resize (see screen recordings).